org.civicrm.hremergency/CRM/Hremergency/Upgrader/Steps/1003.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+trait CRM_Hremergency_Upgrader_Steps_1003 {
+
+  /**
+   * Makes some emergency contact fields required.
+   */
+  public function upgrade_1003() {
+    $params = ['custom_group_id' => 'Emergency_Contacts'];
+    $customFields = civicrm_api3('CustomField', 'get', $params)['values'];
+
+    $shouldBeRequired = [
+      'Name',
+      'Mobile_number',
+      'Relationship_with_Employee',
+      'Dependant_s_',
+    ];
+
+    foreach ($customFields as $customField) {
+      if (!in_array($customField['name'], $shouldBeRequired)) {
```

Comment:
Why not 

> if (in_array($customField['name'], $shouldBeRequired)) {
     $customField['is_required'] = TRUE;
     civicrm_api3('CustomField', 'create', $customField);
}

return TRUE;

Then the code will be a bit more conpact and we would save one or two lines.

Reply:
Fair enough, usually when I start writing code I like to have early returns because the alternative if a possibility of nesting more and more `if` statements, but in this case it won't get more complicated so having the api call in the if block is just as neat.